src/EditorFeatures/CSharpTest/EditAndContinue/SyntaxUtilitiesTests.cs

```diff
@@ -59,5 +59,96 @@ static void Main(string[] args)
 ";
             VerifySyntaxMap(source1, source2);
         }
+
+        [Fact]
+        public void FindLeafNodeAndPartner1()
+        {
+            var leftRoot = SyntaxFactory.ParseSyntaxTree(@"
+using System;
+
+class C
+{
+    public void M()
+    {
+        if (0 == 1)
+        {
+            Console.WriteLine(0);
+        }
+    }
+}
+").GetRoot();
+            var leftPosition = 121; // 0 within Console.WriteLine(0)
```

Comment:
>121 [](start = 31, length = 3)

Can we find the position rather than using a constant? (Is this constant correct when newlines are represented differently?) Perhaps:
```
leftPosition = leftRoot.GetRoot().DescendantNodes().
    OfType<LiteralExpressionSyntax>().
    ElementAt(3).
    Position;
```
Same comment for other tests here and in VB.